src/components/offers/list/OffersView.js

```diff
@@ -11,11 +11,13 @@ import {SorterWrapper} from "../sorter/SorterWrapper";
 import {ColumnAndDetailsLayout} from "../../commons/layouts/ColumnAndDetailsLayout";
 import {StandardViewAndFilterLayout} from "../../commons/layouts/StandardViewAndFilterLayout";
 import CenteredCircularProgress from "../../commons/CenteredCircularProgress";
+import Swal from "sweetalert2";
 
 export const OffersView = (props) => {
 
     const [selectedOffer, setSelectedOffer] = useState(null)
-    const [offers, setOffers] = useState([])
+    const [offers, setOffers] = useState(null)
```

Comment:
Null raczej nie przejdzie, jeśli po tym mapujemy. Wywali się zanim się zdąży zfetchowac

Reply:
Po to jest ten `null`, żeby wiedzieć, że nic jeszcze nie zostało zfetchowane, po pusta arrayka jest kłamstwem, no ale mogę zmienić na flagę co by było ładniej

Reply:
W sensie tu chodzi o to, że ok, jest sobie nullem, ale wiele razy było tak, że potem ktoś sobie zapomni, czy coś i jeb "offers.map is not a function" -> a debuggowanie tego błędu nie jest przyjemne.